selectrum-prescient.el

```diff
@@ -174,9 +174,29 @@ buffer. It does not affect the default behavior (determined by
 (selectrum-prescient-create-and-bind-toggle-command fuzzy "f")
 (selectrum-prescient-create-and-bind-toggle-command initialism "i")
 (selectrum-prescient-create-and-bind-toggle-command literal "l")
+(selectrum-prescient-create-and-bind-toggle-command literal-prefix "P")
 (selectrum-prescient-create-and-bind-toggle-command prefix "p")
 (selectrum-prescient-create-and-bind-toggle-command regexp "r")
 
+(defun selectrum-prescient-toggle-char-fold ()
+  "Toggle character folding in the current Selectrum buffer.
+
+See the customizable variable `prescient-use-char-folding'."
+  (interactive)
+  ;; Make sure that this change is local only to the current Selectrum
+  ;; buffer.
+  (setq-local prescient-use-char-folding
+              (not prescient-use-char-folding))
+  ;; After changing `prescient-use-char-folding', tell the user
+  ;; the new value and update Selectrum's display.
```

Comment:
I think those comments can be removed, the code tells it already ;)

Reply:
Oh I just noticed that those comments are similar to the ones used in the macro for creating other toggle commands. I'm not familiar with prescients code. The comments don't harm but I feel they don't need to be added everywhere where this is done.

Reply:
I removed the comments.